backend/src/main/java/org/sefglobal/core/model/Certificate.java

```diff
@@ -0,0 +1,23 @@
+package org.sefglobal.core.model;
+
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.util.Date;
+
+@Entity
+@Table(name = "certificate")
+@Getter(AccessLevel.PUBLIC)
+@Setter(AccessLevel.PUBLIC)
+public class Certificate extends AuditModel{
```

Comment:
Why does this class `extends AuditModel`. Since we are not storing any audit data I don't think this is necessary.